docs/services/data/dataset.md

```diff
@@ -41,26 +41,7 @@ To interact with datasets programmatically, use the [data client API](/appendix/
 
 {{< readfile "/static/include/app/apis/generated/dataset-table.md" >}}
 
-## Sample dataset
```

Comment:
Yes because it's already in the guide. I don't know how it's useful? It's sooo easy to create a dataset or get one from somewhere else anyway no? It doesn't feel like reference content to me.

Okay yeah I'll add back an image.

Reply:
How would you go about getting a dataset from somewhere else? Can you walk me through it?

Reply:
Do you maybe mean an unlabelled dataset is easy to create?

Reply:
Idk--aren't there things on the internet you could use?

Reply:
Yes I suppose I do mean it's very easy to create an unlabeled one, and not that hard to label a small dataset. In any case, I'm not sure how much usefulness this sample dataset has since it's not applicable to most projects.

Reply:
They are probably only useful in the context of training but let's ask Jack what he thinks about whether we should provide them here.